superset/models/sql_lab.py

```diff
@@ -99,6 +99,7 @@ class Query(Model, ExtraJSONMixin):
     end_time = Column(Numeric(precision=20, scale=6))
     end_result_backend_time = Column(Numeric(precision=20, scale=6))
     tracking_url = Column(Text)
+    json_metadata = Column(Text)
```

Comment:
Can you use the [ExtraJSONMixin](https://github.com/apache/superset/blob/231716cb50983b04178602b86c846b7673f9d8c3/superset/models/helpers.py#L472) to be consistent with other models?